### ISSUE TITLE
konstraint -v is blank

### ISSUE THREAD

**Issue by zephyrdb** [plexsystems/konstraint#559]
When built as recommended in the README ('go install github.com/plexsystems/konstraint@latest'), konstraint -v has a blank version string.  For it to work correctly, I need to do something like this (based on the Makefile):

go install \
    -ldflags="-s -w -X 'github.com/plexsystems/konstraint/internal/commands.version=${KONSTRAINT_VERSION}'" \
    "github.com/plexsystems/konstraint@${KONSTRAINT_VERSION}"

Could this be made to work as described in the README?

**Comment by zephyrdb** [comment]
Hm, perhaps this is relevant https://github.com/golang/go/issues/50603

**Comment by jalseth** [comment]
Yes, that's a limitation of the current approach. An alternative approach would be to have the version baked into the source, which could optionally be overridden with the linker flags.

I'd welcome a PR.
